### DISCUSSION TITLE
Defenseless player protection

### DISCUSSION THREAD

**Discussion by Eilrahcrrod** [GriefPrevention/GriefPrevention#1071] [Q&A] [answered]
I'm working on a pvp server, and one of the modes is just hand-to-hand combat, but the plugin says "you can't hurt defenseless players". I've looked around the config multiple times and haven't found anything that may be able to fix it.  Is there a way to disable that?

**Comment by RoboMWM** [comment, marked as answer]
https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#pvp-configuration `PvP.ProtectFreshSpawns`
